DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/AdminFieldsController.cs

```diff
@@ -106,6 +121,11 @@ public IActionResult EditAdminField(EditAdminFieldViewModel model, string action
         [ServiceFilter(typeof(RedirectEmptySessionData<EditAdminFieldData>))]
         public IActionResult EditAdminFieldBulk(int customisationId, int promptNumber)
         {
+            if (!VerifyAdminUserCanAccessCourse(customisationId))

```

Comment:
ditto about POST

Reply:
added the service filter

Reply:
Added a check in the controller because of the issues using the Service filter here